app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  def self.last_comments(post)
+    return unless post
```

Comment:
- [ ] the posts has many relationship with comment and like should be set in this class

Reply:
Hi @mikethreels, thanks for your review. I'll apply those changes.